runtime/nls/cfre/cfrerr.nls

```diff
@@ -1668,3 +1668,56 @@ J9NLS_CFR_ERR_INIT_ON_VALUE_CLASS.explanation=Please consult the Java Virtual Ma
 J9NLS_CFR_ERR_INIT_ON_VALUE_CLASS.system_action=The JVM will throw a verification or class loading-related exception such as java.lang.ClassFormatError.
 J9NLS_CFR_ERR_INIT_ON_VALUE_CLASS.user_response=Contact the provider of the class file for a corrected version.
 # END NON-TRANSLATABLE
+
+J9NLS_CFR_ERR_MAJOR_VERSION2=bad major version; class=%2$.*1$s, offset=%3$u, major.minor=%4$u.%5$u, maximum major value=%6$u.0
+# START NON-TRANSLATABLE
+J9NLS_CFR_ERR_MAJOR_VERSION2.sample_input_1=4
+J9NLS_CFR_ERR_MAJOR_VERSION2.sample_input_2=main
+J9NLS_CFR_ERR_MAJOR_VERSION2.sample_input_3=6
+J9NLS_CFR_ERR_MAJOR_VERSION2.sample_input_4=62
+J9NLS_CFR_ERR_MAJOR_VERSION2.sample_input_5=0
+J9NLS_CFR_ERR_MAJOR_VERSION2.sample_input_6=61
+J9NLS_CFR_ERR_MAJOR_VERSION2.explanation=The major version in the classfile is unsupported on the VM rejecting the classfile.
+J9NLS_CFR_ERR_MAJOR_VERSION2.system_action=The JVM will throw a verification or classloading related exception such as java.lang.ClassFormatError.
+J9NLS_CFR_ERR_MAJOR_VERSION2.user_response=Contact the provider of the classfile for a corrected version.
+
```

Comment:
Please remove this blank line, and others preceding `# END NON-TRANSLATABLE`, also the repeat blank line 1686.